ffi/src/sspi/sec_winnt_auth_identity.rs

```diff
@@ -348,6 +354,50 @@ pub unsafe fn auth_data_to_identity_buffers_w(
     }
 }
 
+#[cfg(feature = "scard")]
+fn collect_smart_card_creds(username: &[u8], password: &[u8]) -> Result<SmartCardIdentityBuffers> {
+    if username.contains(&b'@') {
+        info!("Trying to collect smart card creds...");
+        use winscard::SmartCardInfo;
+
+        use crate::sspi::utils::raw_wide_str_trim_nulls;
+        use crate::utils::str_encode_utf16;
+        use crate::winscard::scard_context::{DEFAULT_CARD_NAME, MICROSOFT_DEFAULT_CSP};
```

Comment:
style: move at the beginning of the function

rationale: it's useful to have imports local to the function, but for readability they should be at the top (not mixed with logic)